scripts/seedDb.mjs

```diff
@@ -69,10 +69,16 @@ async function recreateLeagueData(leagueKeyPrefix, dataRepo) {
 
 
 async function recreateLeagueConfigurationData(leagueConfigurationKey, dataRepo) {
+    let leagueConfigurationCursor = await redis.scan("0", {match: leagueConfigurationKey+"*"});
+    let leagueConfigurationKeys = leagueConfigurationCursor[1];
+    let cursorStart = leagueConfigurationCursor[0];
+    while(cursorStart !== "0"){
+        leagueConfigurationCursor = await redis.scan(cursorStart, {match: leagueConfigurationKey+"*"});
+        leagueConfigurationKeys = leagueConfigurationKeys.concat(leagueConfigurationCursor[1]);
+        cursorStart = leagueConfigurationCursor[0];
+    }
 
-    const leagueConfigurationCursor = await redis.scan("0", {match: leagueConfigurationKey+"*"})
-
-    for (const aKey of leagueConfigurationCursor[1]) { //list of keys
+    for (const aKey of leagueConfigurationKeys) { //list of keys
```

Comment:
As discussed offline we don't really need to do a scan here because this is a point read/write. Having said that we will need both a way to delete the existing keys as well as a way to "update" keys (or more likely delete them and rewrite them) to update status.